app/presenters/prior_authority/tasks/case_and_hearing_detail.rb

```diff
@@ -11,28 +11,20 @@ def path
         end
       end
 
-      def completed?
-        results = forms_for_completion_hash.each_with_object([]) do |(_k, form_hash), arr|
-          arr << form_hash[:form].build(application).valid? if form_hash[:required]
-        end
-        results.all?(true)
+      def completed?(_rec = record, _form = nil)
+        required_forms.all? { |form| super(record, form) }
```

Comment:
neat, but would it be clearer if the usable argument is used for consistency?

```rb
```
```suggestion
      def completed?(rec = record, _form = nil)
        required_forms.all? { |form| super(rec, form) }
```